tools/sgspanneraipgo/tools.go

```diff
@@ -0,0 +1,24 @@
+package sgspanneraipgo
+
+import (
+	"context"
+	"os/exec"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const (
+	name    = "spanner-aip"
+	version = "v0.61.0"
+)
+
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	sg.Deps(ctx, PrepareCommand)
+	return sg.Command(ctx, sg.FromBinDir(name), args...)
+}
+
+func PrepareCommand(ctx context.Context) error {
```

Comment:
you generally want to do this with the version that's specified in your `go.mod` to make sure you have compatible runtime with code generated APIs

Reply:
not sure it adds enough value to bring this in here, given that it'll lead to code generator and runtime falling out of sync in places that use this helper

Reply:
aha, so that's the reason. I was about to write an explanation, see in slack